gap/fp/word.gi

```diff
@@ -0,0 +1,25 @@
+#############################################################################
+##
+##  word.gi
+##  Copyright (C) 2020                                      James D. Mitchell
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+InstallMethod(WordToString, "for a string and a list",
+[IsString, IsList],
+function(alphabet, word)
+  local str, i, j;
+   j := Maximum(word);
```

Comment:
Not important, but you could just use `Maximum(word)` in the following line instead of defining `j`